### PR TITLE
Updated citation and license to reflect expanded GBIF dataset

### PR DESCRIPTION
**Pull request by timrobertson100**
*Description of changes:*

We have increased the amount of data in the GBIF snapshots, so the license changed and we would like to draw attention to the DOI in the citation as recommended by Erin. 

Thank you

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by timrobertson100** [comment]
Could you please take a look at this @chueatwork?
Many thanks

File: datasets/gbif.yaml [hunk 1]
```diff
@@ -15,7 +15,8 @@ Tags:
   - bioinformatics
   - conservation
   - life sciences
-License: "This dataset is available under a [CC-BY](https://creativecommons.org/licenses/by/4.0/) license and with the GBIF [terms of use](https://www.gbif.org/terms). Please refer to the GBIF [citation guidelines](https://www.gbif.org/citation-guidelines) when using this dataset."
+License: "This dataset is available under a [CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) license and with the GBIF [terms of use](https://www.gbif.org/terms). Please refer to the GBIF [citation guidelines](https://www.gbif.org/citation-guidelines) when using this dataset and include the relevant DOI found from [the table](https://www.gbif.org/occurrence-snapshots)."
```
Comment (chueatwork) [inline datasets/gbif.yaml:18]: Suggest shortening License field to:

"[CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) and under the GBIF [terms of use](https://www.gbif.org/terms)"

File: datasets/gbif.yaml [hunk 2]
```diff
@@ -15,7 +15,8 @@ Tags:
...
+Citation: "`DOI` (Please include the relevant DOI found from [the table](https://www.gbif.org/occurrence-snapshots))"
```
Comment (chueatwork) [inline datasets/gbif.yaml:19]: @timrobertson100 thanks for the ping! So it looks like there are citation guidelines in the license. Knowing also that the Citation field will be appended to the default language citing the data retrieval from Registry of Open Data, what if we rephrased Citation field as:

In addition, include the relevant DOI found from [the table](https://www.gbif.org/occurrence-snapshots)). For more information on how to cite GBIF datasets, please refer to the GBIF [citation guidelines](https://www.gbif.org/citation-guidelines).

Comment (chueatwork) [reply datasets/gbif.yaml:19]: @timrobertson100 let me know what you think of these suggestions!

Erin

Comment (timrobertson100) [reply datasets/gbif.yaml]: Thanks for the suggestions @chueatwork - I've applied these in a subsequent commit.

**Comment by chueatwork** [comment]
Looks great @timrobertson100 , can you please merge upstream changes to your branch? Then I can merge this commit!

**Comment by timrobertson100** [comment]
Thanks, @chueatwork 
rebased and squashed

**Comment by timrobertson100** [comment]
I've rebased again @chueatwork 
Can you please merge this?

**Comment by chueatwork** [comment]
All set. Thanks @timrobertson100 !

**Comment by timrobertson100** [comment]
Thanks @chueatwork 
have a nice Thursday!

**Pull request merged**